crates/burn-tensor/src/tensor/api/int.rs

```diff
@@ -53,7 +53,7 @@ where
     pub fn one_hot(self, num_classes: usize) -> Tensor<B, 2, Int> {
         check!(TensorCheck::one_hot_tensor(self.clone(), num_classes));
         let [num_samples] = self.dims();
-        let indices = self.unsqueeze();
+        let indices = self.unsqueeze_dim(1);
         let values = indices.ones_like();
         Tensor::zeros([num_samples, num_samples], &indices.device()).scatter(1, indices, values)
```

Comment:
Hey hey, yes thank you for changing this. Only realized later and was trying push the fix on another PR.
Forget not to also change this :rocket: 
```suggestion
        Tensor::zeros([num_samples, num_classes], &indices.device()).scatter(1, indices, values)
```
Just so I know in the future, where are these tensor operations tested?
Thanks! :star2:

Reply:
I don't know, first commit here ;) Ah, thanks for the fix, had it in my example, but forgot it here.

Reply:
Ah right, hopefully the reviewer will answer it. Thanks for changing it!

Reply:
@tsanona Tests for this are in `crates/burn-tensor/src/tests/ops/squeeze.rs` (both squeeze and unsqueeze are in here).